packages/material-ui-system/src/breakpoints.test.js

```diff
@@ -28,3 +28,33 @@ describe('breakpoints', () => {
     });
   });
 });
+
+describe('handleBreakpoints', () => {
```

Comment:
This is a private module. Could we reorganize the test case to cover a public API that reproduces the bug? This is important because our current focus with the system is to make it works and right, fast will come after and likely require to reorganize the internal API.

Reply:
@oliviertassinari Just one public api it's worth? Because I reproduced it at least on two APIs (spacing, flexbox).

Reply:
Ideally https://github.com/mui-org/material-ui/blob/c795ca0822e2816a630a5654d5471fa04521d23c/packages/material-ui-system/src/flexbox.js#L4